c/src/ml-api-inference-single.c

```diff
@@ -830,6 +830,36 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
     goto error;
   }
 
+  if (nnfw == ML_NNFW_TYPE_NNTR_INF) {
+    if (!in_tensors_info || !out_tensors_info) {
+      if (!in_tensors_info) {
+        ml_tensors_info_h in_info;
+        status = ml_tensors_info_create (&in_info);
+        if (status != ML_ERROR_NONE) {
+          goto error;
+        }
+
+        status = ml_tensors_info_set_count (in_info, 0);
```

Comment:
This will make error with #102. Let's remove this block as create() will set num-tensors 0.

Reply:
Thank you! removed!